packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I this related to NPM? Seems like a file for development environment. If that's true, should be removed.